app/javascript/packs/body.jsx

```diff
@@ -0,0 +1,183 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import PropTypes from 'prop-types'
+import ReactPaginate from 'react-paginate';
+import Timelogs from './timelogs.jsx'
+import NewTimelog from './new_timelog.jsx'
+
+class Body extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = { timelogs: [], trello_cards: [], page: 0, pageCount: 1, perPage: 3, start_time: 0, end_time: 0 };
```

Comment:
You should follow https://github.com/airbnb/javascript rules.

For readability and git maintainability.
```
this.state = {
  some: 'some',
}
```